sim/scenarios/nat/README.md

```diff
@@ -0,0 +1,24 @@
+# NAT Rebinding
```

Comment:
My recommendation here follows what I wrote in the interop runner PR -- call this Port Rebinding. NAT rebinding is too generic and doesn't capture precisely what this scenario does.